create_hapi_k8s_postgres.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Load environment variables from .env-k8s-sample
+source .env-k8s-sample
+
+# Check if required environment variables are set
+: "${CLOUD_SQL_INSTANCE:?Need to set CLOUD_SQL_INSTANCE}"
+: "${DATABASE_NAME:?Need to set DATABASE_NAME}"
+: "${DATABASE_PASSWORD:?Need to set DATABASE_PASSWORD}"
+
+# 1. Authentication (if necessary)
+gcloud auth application-default login
+
+# 2. Create Cloud SQL instance
+gcloud sql instances create ${CLOUD_SQL_INSTANCE} \
+  --database-version=POSTGRES_14 \
```

Comment:
Should the version of Postgres be configurable, or will version 14 work for users?

Reply:
good idea.